dspace-spring-rest/src/test/java/org/dspace/app/rest/RelationshipRestRepositoryIT.java

```diff
@@ -190,4 +207,270 @@ public void findAllRelationshipTest() throws Exception {
+        Collection col2 = CollectionBuilder.createCollection(context, child1).withName("Collection 2").build();
+        Collection col3 = CollectionBuilder.createCollection(context, child1).withName("OrgUnits").build();
+
+        Item author1 = ItemBuilder.createItem(context, col1)
+                                  .withTitle("Author1")
+                                  .withIssueDate("2017-10-17")
+                                  .withAuthor("Smith, Donald")
+                                  .withRelationshipType("Person")
+                                  .build();
+
+        Item publication = ItemBuilder.createItem(context, col3)
+                                      .withTitle("Publication1")
+                                      .withAuthor("Testy, TEst")
+                                      .withIssueDate("2015-01-01")
+                                      .withRelationshipType("Publication")
+                                      .build();
+
+        RelationshipType isAuthorOfPublicationRelationshipType = relationshipTypeService
+            .findbyTypesAndLabels(context, entityTypeService.findByEntityType(context, "Publication"),
+                                  entityTypeService.findByEntityType(context, "Person"),
+                                  "isAuthorOfPublication", "isPublicationOfAuthor");
+
+
+
+        EPerson user = ePersonService.create(context);
+        user.setFirstName(context, "first");
+        user.setLastName(context, "last");
+        user.setEmail("testazeazeazezae@email.com");
+        user.setCanLogIn(true);
+        user.setLanguage(context, I18nUtil.getDefaultLocale().getLanguage());
+        ePersonService.setPassword(user, password);
+        // actually save the eperson to unit testing DB
+        ePersonService.update(context, user);
+        context.setCurrentUser(user);
+
+        String token = getAuthToken(user.getEmail(), password);
+
+        MvcResult mvcResult = getClient(token).perform(post("/api/core/relationships")
+                                                           .param("relationshipType",
+                                                                  isAuthorOfPublicationRelationshipType.getID()
+                                                                                                       .toString())
+                                                           .contentType(MediaType.parseMediaType("text/uri-list"))
+                                                           .content(
+                               "https://localhost:8080/spring-rest/api/core/items/" + publication.getID() + "\n" +
+                               "https://localhost:8080/spring-rest/api/core/items/" + author1.getID()))
+                                              .andExpect(status().isForbidden())
+                                              .andReturn();
+
+    }
+
+    @Test
+    public void putRelationshipAccess() throws Exception {
```

Comment:
It looks like we only have `PUT` tests for Administrators?  I thought that anyone with WRITE access (to one of the Items) should also be able to update the relationship?  We should also have a test verifying that someone without WRITE access will receive a Forbidden error (see the `create*` tests you made above)

Reply:
I think we never discussed the `PUT` or `DELETE` rights in detail. Maybe that's a good discussion for the next meeting.
If you have a relationship between items a and b, and want to update it to a relationship between a and c, which permissions would you need?

Reply:
The permissions has been updated and an IT has been created